Palo Alto Networks/paloalto/_meta/fields.yml

```diff
@@ -0,0 +1,24 @@
+action.target:
+  description: 'Target action'
+  name: action.target
+  type: keyword
+geo.country_iso_code:
+  description: "Country ISO code"
+  name: geo.country_iso_code
+  type: keyword
+http.response.content_type:
```

Comment:
Maybe `http.response.mime_type` built in field ?

Reply:
I created a custom field for this one. `http.response.mime_type` doesn't seem to be OK according to the doc